packages/datagateway-table/src/state/app.types.tsx

```diff
@@ -1,38 +1,14 @@
 import { ThunkAction } from 'redux-thunk';
 import { AnyAction } from 'redux';
-import { Order, Filter, Entity, DownloadCartItem } from 'datagateway-common';
-import {
-  FeatureSwitches,
-  URLs,
-  BreadcrumbSettings,
-} from './actions/actions.types';
-import { RouterState } from 'connected-react-router';
+import { DGCommonState } from 'datagateway-common';
+import { FeatureSwitches, BreadcrumbSettings } from './actions/actions.types';
 
 export interface DGTableState {
-  facilityName: string;
-  sort: {
-    [column: string]: Order;
-  };
-  filters: {
-    [column: string]: Filter;
-  };
-  data: Entity[];
-  totalDataCount: number;
-  investigationCache: EntityCache;
-  datasetCache: EntityCache;
-  cartItems: DownloadCartItem[];
-  allIds: number[];
-  loading: boolean;
-  downloading: boolean;
-  error: string | null;
-  dataTimestamp: number;
-  countTimestamp: number;
-  allIdsTimestamp: number;
   res?: ApplicationStrings;
   features: FeatureSwitches;
-  urls: URLs;
   breadcrumbSettings: BreadcrumbSettings;
   settingsLoaded: boolean;
+  facilityName: string;
```

Comment:
I thought facilityName was moved to `common`?

Reply:
yes it was ;)